src/emqx_mgmt_cli.erl

```diff
@@ -101,7 +111,7 @@ mgmt(["list"]) ->
 
 mgmt(_) ->
     emqx_cli:usage([{"mgmt list",                   "List Applications"},
-                    {"mgmt insert <AppId> <Name>",   "Add Application of REST API"},
+                    {"mgmt insert <AppId> <Name> [<Secret>]",   "Add Application of REST API"},
```

Comment:
What about `mgmt insert <AppId> <Name> <Secret>`

Reply:
Other cli commands have use square brackets for optional arguments. Correct if I'm wrong.

Reply:
We have discussed the feature you implement, there are still controversy in our team. Anyway, appreciate your work and thanks for you contribution, we would give you feedback soon.